packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts

```diff
@@ -1,5 +1,5 @@
 import { PluginInstallation } from "@amplication/code-gen-types";
-import { createLog } from "@amplication/data-service-generator/create-log";
+import { createLog } from "@amplication/data-service-generator";
```

Comment:
How the other changes helped you with this import?

Reply:
Its blocking us to make the same mistake again

Reply:
I mean, before the changes you have applied, you wouldn't be able to import `createLog ` from `@amplication/data-service-generator`. How these changes (in the other files) helped you to import it like that?

Reply:
https://github.com/amplication/amplication/pull/4978/files#diff-6f1199c262f20ad5f5baec5d4f927a8c6f8162389220df492b17fac5945dce68